README.md

```diff
@@ -139,38 +147,39 @@ Let's look at an example:
 ```tsx
 import { AppProps } from "next/app";
 import { Provider, cacheExchange, fetchExchange, ssrExchange } from "urql";
+import { SaleorAuthProvider, useAuthChange } from "@saleor/auth-sdk/react";
 
-import { SaleorAuthProvider, useAuthChange, useSaleorAuthClient } from "@saleor/auth-sdk/react";
-import { useUrqlClient } from "@saleor/auth-sdk/react/urql";
+const saleorApiUrl = "<your Saleor API URL>";
 
-const SaleorURL = "<your Saleor API URL>";
-
-export default function App({ Component, pageProps }: AppProps) {
-  const useSaleorAuthClientProps = useSaleorAuthClient({
-    saleorApiUrl: SaleorURL,
-  });
+const saleorAuthClient = createSaleorAuthClient({ saleorApiUrl });
 
-  const { urqlClient, reset, refetch } = useUrqlClient({
-    url: SaleorURL,
-    fetch: useSaleorAuthClientProps.saleorAuthClient.fetchWithAuth,
+const makeUrqlClient = () =>
+  createClient({
+    url: saleorApiUrl,
+    fetch: saleorAuthClient.fetchWithAuth,
     exchanges: [cacheExchange, fetchExchange],
   });
 
+export default function App({ Component, pageProps }: AppProps) {
+  const [urqlClient, setUrqlClient] = useState<Client>(makeUrqlClient());
```

Comment:
Maybe we should add a comment with a link to the issue that describes this pattern?
```suggestion
  // https://github.com/urql-graphql/urql/issues/297#issuecomment-504782794
  const [urqlClient, setUrqlClient] = useState<Client>(makeUrqlClient());
```